### PR TITLE
#164734652  Add UI for upload an image

### PR DESCRIPTION
**Pull request by misocho**
#### What does this PR do?
Enable a user to upload an image from local storage
#### Description of the task to be completed
Enable a user to upload an image from local storage
#### How should this be manually tested?
* Clone the repo `git clone https://github.com/martinMutuma/def-politico.git`
* Checkout to this branch `ft/164734652-ui-upload-image`
* Navigate to the profile page
* Click on the edit icon
* Notice a drop down box with the `upload an image button`
#### Any background context:
N/A
#### What are the relevant pivotal tracker stories?
#164734652
#### Screenshots
image
image

### CODE REVIEW

**Comment by Tevinthuku** [comment]
I see the drop down works as expected.
You can add an `accept` field to the input so that it can only accept to upload images like so
```html
      <input accept="image/*"  type="file" />
```

File: UI/profile.html [hunk 1]
```diff
@@ -30,9 +31,28 @@
...
+                                             Edit   
+                                            </div>
+                                    </div>
+                            </summary>
+                            <details-menu class="dropdown-menu">
+                              <label for="input-field">Upload an image</label>
+                              
+                              <button id="rm-btn">Remove image</button>
+                            </details-menu>
+                            <input type="file" id="input-field">
```
Comment (Tevinthuku) [inline UI/profile.html:52]: As it currently is someone can even pick a video. The accept property can help in filtering out unwanted file types from the User's filesystem. Apart from this everything else works as you had intended it to. Great job :+1: 
```html
      <input accept="image/*"  type="file" />
```

**Comment by misocho** [comment]
Thanks... just made the changes @Tevinthuku

**Review by Tevinthuku — COMMENTED** [review]
There's an empty `UI/js/profile.js` file, were you intending on having anything inside it?

**Comment by Tevinthuku** [comment]
I've re-requested a review, that was an accident :laughing:

**Review by Tevinthuku — APPROVED** [review]

**Comment by miritih** [comment]
Kindly have a look at the failing checks @misocho

**Comment by misocho** [comment]
@miritih The failing checks are due to an error in codacy. It can't analyse gh-pages branch.

**Comment by misocho** [comment]
@miritih 
image

**Review by miritih — APPROVED** [review]

**Pull request merged**